### PR TITLE
Improved documentation for _preValidatePurchase method in Crowdsale

### LINKED ISSUE #1096: Add examples of extending _preValidatePurchase in Crowdsale.sol
While reading through the `Crowdsale.sol` smart contract, I am wondering if someone can explain to me what this portion of the comments mean:

> Use super to concatenate validations.

What can be a reason for concatenating validations, and can we have an example of doing this?

https://github.com/OpenZeppelin/openzeppelin-solidity/blob/92b695f2fb258a5669181d1d8cf8e73aa756f22c/contracts/crowdsale/Crowdsale.sol#L110-L123

Thank you!

### PR DESCRIPTION
**Pull request by dougiebuckets**
Fixes #1096 

# 🚀 Description

Improved documentation for _preValidatePurchase in Crowdsale. Hopefully this makes it more clear how one can use `super` in contracts that inherit from Crowdsale in order to extend their validations.

- [x] 📘 I've reviewed the [OpenZeppelin Contributor Guidelines](../blob/master/CONTRIBUTING.md)
- [x] 🎨 I've run the JS/Solidity linters and fixed any issues (`npm run lint:all:fix`).

### CODE REVIEW

**Comment by nventuro** [comment]
Thanks!

I noticed however you rolled back two typo fixes, and changed `SafeERC20` for the standard one, was that intentional or did you add those changes to this PR by mistake?

**Comment by dougiebuckets** [comment]
Thanks for the heads up! Not sure how I did that? Should be all set now.

File: contracts/crowdsale/Crowdsale.sol [hunk 1]
```diff
@@ -110,7 +110,10 @@ contract Crowdsale {
   // -----------------------------------------
 
   /**
-   * @dev Validation of an incoming purchase. Use require statements to revert state when conditions are not met. Use super to concatenate validations.
+   * @dev Validation of an incoming purchase. Use require statements to revert state when conditions are not met. Use `super` in contracts that inherit from Crowdsale to extend their validations.
+   * @dev Example from CappedCrowdsale.sol's _preValidatePurchase method:
```
Comment (frangio) [inline contracts/crowdsale/Crowdsale.sol]: Nit: The second line shouldn't have a `@dev` tag. It somehow messes up Solidity's NatSpec parser. We changed this across OpenZeppelin in #995.

Comment (dougiebuckets) [reply contracts/crowdsale/Crowdsale.sol]: Thanks! Didn't realize that. Fixed!

**Pull request merged**